Offseason2022/src/main/java/frc/robot/commands/ClimberFullClimb.java

```diff
@@ -69,9 +71,12 @@ public ClimberFullClimb(Climber climber, XboxController gamePad, Button button)
             new WaitCommand(m_rotateRetractL4Time), 
             new ClimberTimerOverride(climber, gamePad, button)
         ),
-        new Climber7ClimbToL4(climber)
-        // @formatter:on 
+        new Climber7ClimbToL4(climber),
+        new InstantCommand(climber::timerPrint, climber)
     );
+        // @formatter:on 
+    ;

```

Comment:
Not sure what this hanging semi-colon is for, so let's clean it up. And move the @formatter:on directive right after the InstantCommand (this makes both directives within the AddCommands brackets). Works either way, but these would be nice cleanups.

Otherwise it's all functional and looks good.